### PR TITLE
Docs: Setting up VTEX account for FastStore

### PR DESCRIPTION
**Pull request by Mariana-Caetano**
## What's the purpose of this pull request?

This PR adds the first of five docs about integrating the FastStore project to the VTEX platform.

P.s.: This documentation will be moved to the [go-live track](https://faststore-site-git-vtex-integration-faststore.vercel.app/docs/go-live/1-setting-up-vtex-account-for-faststore) once we merge the #1765.

🔗 [Preview  ](https://faststore-site-git-docs-setting-up-account-faststore.vercel.app/docs/setting-up-a-vtex-account)

### CODE REVIEW

File: apps/site/pages/docs/setting-up-a-vtex-account.mdx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+1 . Log in to your VTEX account:
+
+<Callout type="info" emoji="ℹ️">
+   Remember to replace the values between curly brackets according to your scenario.
+</Callout>
+
+   ```
+   vtex login {account}
+   ```
```
Comment (lucasfp13) [inline apps/site/pages/docs/setting-up-a-vtex-account.mdx:45]: These whitespaces are causing a break in the positioning of the content.
Removing them solves the problem:

| Before | After |
| --- | --- |
| Screenshot 2023-05-29 at 15 52 49 | Screenshot 2023-05-29 at 16 01 55 |

File: apps/site/pages/docs/setting-up-a-vtex-account.mdx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+
+2. Install the FastStore plugin for the VTEX IO CLI:
+
+   ```
+   vtex plugins install faststore
+   ```
```
Comment (lucasfp13) [inline apps/site/pages/docs/setting-up-a-vtex-account.mdx:52]: Same here regarding the whitespaces

File: apps/site/pages/docs/setting-up-a-vtex-account.mdx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+3. Set up your VTEX account for FastStore by running the following command:
+
+   ```
+   vtex faststore setup
+   ```
```
Comment (lucasfp13) [inline apps/site/pages/docs/setting-up-a-vtex-account.mdx:58]: Same here regarding the whitespaces

**Review by lucasfp13 — COMMENTED** [review]
LGTM, just left some suggestions 🎉

File: apps/site/pages/docs/setting-up-a-vtex-account.mdx [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+   Remember to replace the values between curly brackets according to your scenario.
+</Callout>
+
+   ```
+   vtex login {account}
+   ```
+
+
+2. Install the FastStore plugin for the VTEX IO CLI:
+
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/setting-up-a-vtex-account.mdx:49]:
```suggestion
```

File: apps/site/pages/docs/setting-up-a-vtex-account.mdx [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+## Step by step
+
+You will use the FastStore plugin for the VTEX IO CLI to install, with a single command, all dependencies needed for developing a FastStore website.
+
+1. Log in to your VTEX account:
+
+<Callout type="info" emoji="ℹ️">
+   Remember to replace the values between curly brackets according to your scenario.
+</Callout>
+
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/setting-up-a-vtex-account.mdx:42]:
```suggestion
```

File: apps/site/pages/docs/setting-up-a-vtex-account.mdx [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+   ```
+   vtex login {account}
+   ```
+
+
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/setting-up-a-vtex-account.mdx:47]:
```suggestion
```

File: apps/site/pages/docs/setting-up-a-vtex-account.mdx [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+1. Log in to your VTEX account:
+
+<Callout type="info" emoji="ℹ️">
+   Remember to replace the values between curly brackets according to your scenario.
+</Callout>
+
+   ```
+   vtex login {account}
+   ```
+
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/setting-up-a-vtex-account.mdx:46]:
```suggestion
```

File: apps/site/pages/docs/setting-up-a-vtex-account.mdx [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+
+2. Install the FastStore plugin for the VTEX IO CLI:
+
+   ```
+   vtex plugins install faststore
+   ```
+
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/setting-up-a-vtex-account.mdx:53]:
```suggestion
```

File: apps/site/pages/docs/setting-up-a-vtex-account.mdx [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+3. Set up your VTEX account for FastStore by running the following command:
+
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/setting-up-a-vtex-account.mdx:55]:
```suggestion
```

File: apps/site/pages/docs/setting-up-a-vtex-account.mdx [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+   ```
+   vtex faststore setup
+   ```
+
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/setting-up-a-vtex-account.mdx:59]:
```suggestion
```

File: apps/site/pages/docs/setting-up-a-vtex-account.mdx (hunk 1 above)
Comment (Mariana-Caetano) [reply apps/site/pages/docs/setting-up-a-vtex-account.mdx:45]: Good catch! Thanks, @lucasfp13

Comment (renatamottam) [reply apps/site/pages/docs/setting-up-a-vtex-account.mdx:44]: I love my developers with designer eyes 😍

**Review by renatamottam — APPROVED** [review]
Content looks solid, lew a tiny fix as a suggestion :)

**Review by lucasfp13 — APPROVED** [review]

**Pull request merged**